src/main/java/attendance/controller/AttendanceController.java

```diff
@@ -0,0 +1,117 @@
+package attendance.controller;
+
+import attendance.constant.Holiday;
+import attendance.domain.Attendance;
+import attendance.domain.Attendances;
+import attendance.domain.Crew;
+import attendance.domain.Crews;
+import attendance.file.AttendanceFileReader;
+import attendance.file.AttendanceFileReader.FileContents;
+import attendance.util.DateUtil;
+import attendance.view.InputView;
+import attendance.view.OutputView;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.Locale;
+
+public class AttendanceController {
+
+    private Attendances attendances;
+    private Crews crews;
+
+    public AttendanceController() throws IOException {
+        String path = "src/main/resources/attendances.csv";
+        FileContents fileContents = AttendanceFileReader.read(path);
+        attendances = fileContents.attendances();
+        crews = fileContents.crews();
+    }
+
+    public void run() {
+        while (true) {
+            String inputFunction = InputView.readFunction();
+            try {
+                performFunction(inputFunction);
+                if (inputFunction.equals("Q")) {
```

Comment:
```suggestion
                if ("Q".equals("Q")) {
```
문자열에 대해 equals를 사용할 때는 위와 같이 작성해주는 것이 좋습니다! String 이 null 인 경우 equals 메서드 사용시 NPE 가 발생할 수 있어요

Reply:
평소에 많이 봤는데 그런 이유였군요! 덕분에 새로 알아갑니다!